src/components/organize/journeys/JourneyMilestoneProgress.tsx

```diff
@@ -29,8 +29,10 @@ const JourneyMilestoneProgress = ({
           textTransform: 'lowercase',
         }}
       >
-        {`${percentCompleted}% `}
-        <Msg id="pages.organizeJourneyInstance.complete" />
+        <Msg
+          id="pages.organizeJourneyInstance.percentComplete"
+          values={{ percentComplete }}
+        />
```

Comment:
This is identical to what I did to hack it together just to get it to work locally while I was reviewing. Great!